cost/downsize_instance/downsize-instance.pt

```diff
@@ -1,8 +1,8 @@
-name "Downsize Instances based on CPU and Memory"
+name "Downsize Instances based on CPU and Memory Update"
```

Comment:
I don't understand how `Update` applies in this name.

Reply:
Carry over from testing.  Fixed.  Thanks